templates/fargate.cfn.yml

```diff
@@ -177,6 +177,36 @@ Parameters:
       - prod
     ConstraintDescription: Specify either dev or prod
 
+  LoadBalancerAlarmEvaluationPeriods:
+    Description: The number of periods over which data is compared to the specified threshold 
+    Type: Number
+    Default: 1
```

Comment:
Please update the default to two. Someone suggested a minimum number of two periods in another PR to guard against potentially unstable metrics.